content/docs/0.8.1/deploy/upgrade/longhorn-manager.md

```diff
@@ -26,203 +24,3 @@ helm upgrade longhorn ./longhorn/chart
 
 Next, [upgrade Longhorn engine.](../upgrade-engine)
-
-### Upgrading from v0.6.2 or older version to v0.8.1
-
-#### Migrate PVs and PVCs for the Volumes Launched in v0.6.2 or Older
-
-If a volume is launched and used in Longhorn v0.6.2 or older, the related persistent volumes (PVs) and persistent volume claims (PVCs) are still managed by the old CSI plugin, which will be deprecated in a later Longhorn version.
-
-Therefore, the PVCs and PVs should be migrated to use the new CSI plugin for the volume in Longhorn v0.8.1.
-
-##### Prerequisites
-
-- Longhorn is already upgraded to v0.8.1.
-- The related PVs and PVCs were created in v0.6.2 or older.
-- Each volume is detached and the workloads are down.
-
-##### Migration Steps
-
-1. If you don't know when the volumes were created, find out which volumes need to be migrated by running the following command:
-
-    ```
-    kubectl get pv --output=jsonpath="{.items[?(@.spec.csi.driver==\"io.rancher.longhorn\")].spec.csi.volumeHandle}"
-    ```
-
-2. Shut down the related workloads and detach the volumes. 
-3. Run this script for each volume:
-
-    ```
-    curl -s https://raw.githubusercontent.com/longhorn/longhorn/v0.8.1/scripts/migrate-for-pre-070-volumes.sh |bash -s -- <volume name>
-    ```
-
-    Or run the script for all volumes:
-    ```
-    curl -s https://raw.githubusercontent.com/longhorn/longhorn/v0.8.1/scripts/migrate-for-pre-070-volumes.sh |bash -s -- --all
-    ```
-**Result:** The volumes have been migrated to use the new CSI driver.
-
-##### Migration Failure handling
-
-If the prerequisite of the migration is not satisfied and there is no error log `failed to delete then recreate PV/PVC, users need to manually check the current PVC/PV then recreate them if needed`, the script will do nothing for the PV and PVC. Users can check the migration prerequisites and steps and retry it. 
-
-If the migration fails and the error log mentioned above is printed out, users need to manually handle the migration for the failed volume:
-
-1. Update `spec.persistentVolumeReclaimPolicy` to `Retain` for the PV with this command:
-
-   ```
-   kubectl edit pv <The PV name>
-   ```
-
-2. Delete the PVC and PV with this command:
-
-    ```
-    kubectl delete pvc <The PVC name> && kubectl delete pv <The PV name>
-    ```
-
-3. Use the Longhorn UI to recreate the PV and PVC.
-
-### Upgrading Longhorn Manager from v0.6.2 to v0.7.0
-
-You will need to follow this guide to upgrade the Longhorn manager from v0.6.2 to v0.7.0.
-
-Live upgrades are not supported from v0.6.2 to v0.7.0.
-
-- [Prerequisites](#prerequisites)
-- [Upgrade Longhorn Manager](#upgrade-longhorn-manager)
-  - [Using the Rancher Catalog App](#using-the-rancher-catalog-app)
-  - [Using kubectl](#using-kubectl)
-- [Troubleshooting](#troubleshooting)
-- [Clean up the v0.6.2 CRDs](#clean-up-the-v062-crds)
-- [Rollbacks](#rollbacks)
-
-### Prerequisites
```

Comment:
This part of content is for general upgrade steps.

Reply:
What... I think this part is for `v0.6.2 to v0.7.0` only. See line 87:
```
You will need to follow this guide to upgrade the Longhorn manager from v0.6.2 to v0.7.0.
```

Maybe we can retain some troubleshooting guides here. But we need to make sure they are for the general cases.

Reply:
The steps here has more details than the steps in `### Upgrading Longhorn Manager from v0.7.0+`.

For example, the previous one doesn't mention upgrade using Rancher catalog app or have a list of the components that suppose to be running after the upgrade.

Reply:
I saw there are some updates for this doc. I decide not to modify this part for now.